### ISSUE TITLE
Citation dialog accessibility issues with JAWS (and maybe others)

### ISSUE THREAD

**Issue by adomasven** [zotero/zotero#3981]
Report https://forums.zotero.org/discussion/comment/460767/#Comment_460767

> When you have selected citations, you can navigate between them with the arrow keys, but without spoken feedback. So you do not know, which citation has the focus and you are gong to edit. That is especially important when the order changes because of the sorting citation-option. You can easily open the box to enter page, prefix and suffix. You can not force Jaws to read out the information in the blue rectangle on the screenshot and it is not read out automatically. Sometimes you can get to hear it by JawsKey+B, but not in all cases. You can reach the fields page, prefix and suffix with the arrow keys. You do not get a spoken feedback in which field the cursor is. You cannot control entered text with the arrow keys due to missing feedback from Jaws. It is the same thing with the editor.
